CONTRIBUTING.md

```diff
@@ -1,6 +1,72 @@
-# Contributing
+# Contributing Guide
 
-To contribute to this repository, fork it and send pull requests.
+We happily welcome contributions to this package. We use [GitHub Issues](https://github.com/databricks/databricks-sql-nodejs/issues) to track community reported issues and [GitHub Pull Requests](https://github.com/databricks/databricks-sql-nodejs/pulls) for accepting changes.
```

Comment:
one small issue, the link is to nodejs not go?

Reply:
@susodapop

Reply:
Fixed in [4cc8942](https://github.com/databricks/databricks-sql-go/pull/19/commits/4cc89426c410c12d62537226d0a76fb124ffaa93)